charts/echo/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+appVersion: 0.0.1
+description: Equinix echo is an example echo client / server application.
+name: equinix-echo
```

Comment:
Definitely open to a better demo use case that shows connectivity between cluster and machine if y'all have ideas.

Reply:
We could probably hack something from this list to work: https://www.virtuallyghetto.com/2020/06/interesting-kubernetes-application-demos.html

There is a pacman app that was created for use with federation demos that we could also likely hack up pretty easily for our use: https://github.com/font/k8s-example-apps

Reply:
Looks like there are some interesting ones there. From what I can tell they mostly involve both places running kubernetes, and I think in this demo only the CAPI side will be right?

Reply:
Yeah, I think any would require some modification for sure, but anything that has a database component is something that we could move out of k8s relatively easily for our purposes, I think.

Reply:
@detiber so are you thinking run database component on crossplane provisioned machine and the app on the CAPI cluster?